src/main/java/org/datadog/jenkins/plugins/datadog/DatadogGlobalConfiguration.java

```diff
@@ -392,9 +392,10 @@ public boolean configure(final StaplerRequest req, final JSONObject formData) th
             if(StringUtils.isNotBlank(this.getHostname()) && !DatadogUtilities.isValidHostname(this.getHostname())){
                 throw new FormException("Your hostname is invalid, likely because it violates the format set in RFC 1123", "hostname");
             }
+            
             this.setHostname(formData.getString("hostname"));
-            this.setExcluded(formData.getString("excluded"));
-            this.setIncluded(formData.getString("included"));
+            this.setExcluded(formData.getString("blacklist"));
```

Comment:
Can you add a comment that these variable names are being kept for backward compatibility?